miso-web/src/main/webapp/pages/editUser.jsp

```diff
@@ -189,8 +188,7 @@
               <c:otherwise>
                 <tr>
                   <td>Password:</td>
-                  <td><i>Password change support only available for the 'jdbc' security method. If using LDAP, please change
-                    the user password in your LDAP server.</i></td>
+                  <td><i>Change password using your the method specified by IT.</i></td>
```

Comment:
Ok. Then could you please remove the extraneous "your" (4th word, and also in the changed line above)?